changelogs/fragments/fix_from_xml.yaml

```diff
@@ -0,0 +1,3 @@
+---
+breaking_changes:
```

Comment:
hey i dont think these are breaking changes, could you please update this to minor changes?

Reply:
sure, done.